vpc/service/auth.go

```diff
@@ -39,30 +29,7 @@ type titusVPCAgentServiceAuthFuncOverride struct {
 }
 
 func (vpcService *titusVPCAgentServiceAuthFuncOverride) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
-	peer, ok := peer.FromContext(ctx)
-	if !ok {
-		return ctx, status.Error(codes.Internal, "Could not retrieve peer from context")
-	}
-	if peer.AuthInfo == nil {
-		// TODO: Log this
-		return ctx, errNoAuth
-	}
-
-	tlsInfo, ok := peer.AuthInfo.(credentials.TLSInfo)
-	if !ok {
-		return ctx, fmt.Errorf("Received unexpected authentication type: %s", peer.AuthInfo.AuthType())
-	}
-	sv := tlsInfo.GetSecurityValue().(*credentials.TLSChannelzSecurityValue)
-	cert, err := x509.ParseCertificate(sv.RemoteCertificate)
-	if err != nil {
-		return ctx, errors.Wrap(err, "Cannot parse remote certificate")
-	}
-	_, err = cert.Verify(x509.VerifyOptions{
-		Roots:     vpcService.TitusAgentCACertPool,
-		KeyUsages: []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth},
-	})
-	if err != nil {
-		return nil, errors.Wrap(err, "Unable to verify client cert")
-	}
+	// TODO: Authn happens via the normal TLS verification, here is where
```

Comment:
If we ever move "back" to a Titus Agent specific PKI, we will want to be able to add that root for authz.

Reply:
I don't think we need to move to Titus Agent specific PKI (not that we had that before anyway).
I think we just need:
* A good root of trust for authn (done)
* A regex to look at the CN to look for titus-agent for authz (in #427)